src/App.jsx

```diff
@@ -14,57 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [title, setTitle] = useState('Jam is selected');
```

Comment:
title is not a state. it is just a variable that includes state. it will be recalculated each time selected good is changed

Reply:
Honestly, I can't understand what I need to change?

Reply:
Pls review my solution